### PR TITLE
removed old code

### PR DESCRIPTION
**Pull request by OskarStark**
I am targeting this branch, because this is BC.

Refs #4817 

see: https://github.com/sonata-project/SonataAdminBundle/pull/4817/files/437e907aaa37feb1c83787e3dd2c5346273c6bf8#diff-156ed93418fe946ee724987b790b24ba

### CODE REVIEW

**Comment by kunicmarko20** [comment]
Shouldn't that still be there?

**Comment by OskarStark** [comment]
Not sure, thats why I asked in the related comment 😄

**Comment by kunicmarko20** [comment]
can't even find the `sonata.admin.exporter` service we only have:

https://github.com/sonata-project/SonataAdminBundle/blob/809daea1b3c35424fadcbe12ccb147ce8b1931a5/src/Resources/config/exporter.xml#L4

**Comment by OskarStark** [comment]
I guess we need to keep it!

> can't even find the sonata.admin.exporter service we only have:

it could come from https://github.com/sonata-project/exporter if I am right @greg0ire

**Comment by kunicmarko20** [comment]
I don't think so, not sure but it doesn't seem right that we would have `sonata.admin.ANYTHING` in exporter, maybe I am wrong?

**Comment by jordisala1991** [comment]
Well, maybe it can be added to the xml itself?

**Comment by kunicmarko20** [comment]
It can be added, but the question is, do we even have that service? 😄

**Comment by jordisala1991** [comment]
https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Resources/config/exporter.xml

Isnt it here?

**Comment by kunicmarko20** [comment]
the id is `sonata.admin.admin_exporter` and here we are deprecating `sonata.admin.exporter`?

**Comment by jordisala1991** [comment]
https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Resources/config/core.xml#L57

here

**Comment by kunicmarko20** [comment]
great, lets add deprecation there then?

**Comment by jordisala1991** [comment]
Can you add the deprecated tag on the xml @OskarStark ?

**Comment by OskarStark** [comment]
Done

File: src/Resources/config/core.xml [hunk 1]
```diff
@@ -55,6 +55,8 @@
         </service>
         <!-- NEXT_MAJOR : remove this service -->
         <service id="sonata.admin.exporter" class="Sonata\AdminBundle\Export\Exporter" public="true"/>
+            <deprecated>The service "%service_id%" is deprecated in favor of the "sonata.exporter.exporter" service since version 3.14 and will be removed in 4.0.</deprecated>
+        </services>
```
Comment (jordisala1991) [inline src/Resources/config/core.xml]: this should be service

File: src/Resources/config/core.xml [hunk 2]
```diff
@@ -55,6 +55,8 @@
         </service>
         <!-- NEXT_MAJOR : remove this service -->
         <service id="sonata.admin.exporter" class="Sonata\AdminBundle\Export\Exporter" public="true"/>
```
Comment (jordisala1991) [inline src/Resources/config/core.xml]: and you cant close this one here />

Comment (OskarStark) [reply src/Resources/config/core.xml]: done

**Comment by jordisala1991** [comment]
Thank you @OskarStark

**Pull request merged**